zio-schema/shared/src/main/scala/zio/schema/diff/LCSDiff.scala

```diff
@@ -0,0 +1,154 @@
+package zio.schema.diff
+
+import zio.schema.{ Diff, Differ }
+import zio.{ Chunk, ChunkBuilder }
+
+object LCSDiff {
```

Comment:
Is it possible to make this package private? Or, seeing as how the `apply` method builds a `Differ`, maybe we should place it into the companion object of `Differ`, where developers will look for constructors of `Differ`?